packages/actor-abstract-mediatype-utilities/lib/AbstractMediatypeUtilities.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  DataSources,
+  IDataSource,
+  KEY_CONTEXT_SOURCE,
+  KEY_CONTEXT_SOURCES,
+} from "@comunica/bus-rdf-resolve-quad-pattern";
+import {ActionContext} from "@comunica/core";
+
+/**
+ * A comunica Hash Query Operation Actor.
+ */
+export abstract class AbstractMediatypeUtilities {
```

Comment:
Since all methods in this class are static, and nothing will extend from it, it is not needed to have config files for this. So you can essentially remove the `components` folder in this package.

Also, I still don't really like this name of this class and package. What about `DataSourceUtils` (package name: `util-datasource`) ?